stdlib.sh

```diff
@@ -78,8 +78,14 @@ fi # }}}
                             # improve MacOS compatibility
 #export std_RELEASE="1.4.7" # Fix warnings identified by shellcheck.net, add
                             # std::wordsplit
-export  std_RELEASE="1.5.0" # Add std::inherit, finally make errno functions
+#export  std_RELEASE="1.5.0" # Add std::inherit, finally make errno functions
                             # work!  Set std_ERRNO where appropriate
+#export std_RELEASE="1.6.0" # Added coloured output tags
+#export std_RELEASE="1.6.1"	# Ehnaced colour setup with optionale external
+							# configuration file, parsing and initialisation
+							# 'Section' helpers enriched with key/value pair
+							# parsing functions
+export std__RELEASE="1.7.0"	# Added command execution functions
```

Comment:
Rather than the version numbering being `<major>.<minor>.<change>`, I was more working along the lines of `<api-version>.<major>.<minor>` - so these changes would be better numbered as `1.5.1` (from `1.6.0` and `1.6.1`) and `1.5.2` (from `1.7.0`).